web/js/env.js

```diff
@@ -1,14 +1,15 @@
 const env = (() => {
-    const win = $(window);
+    // UI
     const doc = $(document);
+    const gameBoy = $('#gamebody');
```

Comment:
it's not gameBoy though, can name it gamebody as well :D

Reply:
I renamed it for a better clarity. What does a `gamebody` even mean? `Gameboy` at least something relatable and can be associated with the part of the interface.

Reply:
Oh, forgot that you have to have ™ mark after it (: GameBoy™